src/components/RegionCard/index.tsx

```diff
@@ -0,0 +1,116 @@
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import {
+  Box,
+  Divider,
+  LinearProgress,
+  Paper,
+  Typography,
+  useTheme,
+} from '@mui/material';
+
+import { RegionMetadata } from '@/models';
+
+import styles from './index.module.scss';
+import { Label } from '../elements';
+
+interface RegionCardProps {
+  region: RegionMetadata;
+}
+
+export const RegionCard = ({ region }: RegionCardProps) => {
+  const { length } = region;
+  const theme = useTheme();
+  const progress = [
+    {
+      label: 'Core Ownership',
+      value: region.ownership,
+      color: 'warning',
+    },
+    {
+      label: 'Consumed',
+      value: region.consumed,
+      color: 'success',
+    },
+    {
+      label: 'Current Usage',
+      value: region.task?.usage ?? 0,
+      color: 'primary',
+    },
+  ];
+
+  return (
+    <Paper className={styles.container}>
+      <div className={styles.regionInfo}>
+        <div
+          className={styles.duration}
+          style={{
+            borderColor: theme.palette.grey[200],
+            color: theme.palette.grey[200],
+          }}
+        >
+          <AccessTimeIcon sx={{ fontSize: '1.25em' }} />
+          {`Duration: ${length}`}
+        </div>
+        <Typography variant='subtitle2'>Region 1 </Typography>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '0.5rem',
+            color: theme.palette.grey[200],
+          }}
+        >
+          <Typography variant='h2'>Begin: 5 days ago</Typography>
+          <Typography variant='h2'>End: 15 days later</Typography>
```

Comment:
The same applies here and the rest of the data in this file.

Reply:
same